brev-bestilling-boot/src/main/resources/application.yml

```diff
@@ -5,6 +5,10 @@ security-token-service:
         password: ${PASSWORD_SRV_BRUKER}
         apikey: ${STS_API_KEY}
 
+brevmetadata:
+    endpoint:
```

Comment:
For consistency, security token service bruker har "client" istedenfor "endpoint". Jeg også lurte på veldig mye hvis jeg bruker "endpoint" eller "client" som navn. Men på slutten ended med "clinet" side vi står på client siden.